app/src/forms/email/assets/bodies/submission-unassigned.html

```diff
@@ -0,0 +1,21 @@
+<table border="0" cellpadding="0" cellspacing="0" role="presentation" style="" width="100%">
+  <tr>
+    <td data-slot-container="1" align="center" style="
+      font-size: 0px;
```

Comment:
This line seems strange esp with a 14px font style one child DOM element down. What is it supposed to do?

Reply:
removed that font-size assignment. tested and looks good in emails still.